src/js/site/insightsTableScroll.test.js

```diff
@@ -0,0 +1,75 @@
+import { insightsTableScroll } from './insightsTableScroll';
+
+window.$ = require('src-api/source/javascripts/lib/_jquery.js');
```

Comment:
We shouldn't require jquery from src-api...
I susepct you have written some tests that use `$` and we don't have jquery available in our test runs. So one of two possible fixes are:

1. Make jquery available by installing it as a package and making it importable to jest tests.
2. Use vanilla js dom selectors.

I think `2` makes the most sense here and we can pair on setting this up if you want.

Reply:
That be great!